index.bs

```diff
@@ -3081,16 +3081,12 @@ There are several types of references:
 	<em>playing</em> reference to themselves while they are currently
 	playing.
 
-3. An <dfn>active reference</dfn> for {{AudioWorkletNode}}s
-	whose {{AudioWorkletNode/processorState}}
-	property is set to <code>running</code>.
-
-4. A <em>connection</em> reference which occurs if another
+3. A <em>connection</em> reference which occurs if another
```

Comment:
The change in 3084, 3089, 3099 and 3105 do not seem to be related to the AudioWorklet change. Is this intended?

Reply:
It's just a change in the numbers.  Since the original numbered them in numerical order, and this change removed one of the entries, I decided to keep numerical order, without a gap.

Should probably just make them all the same number so we never have to renumber.  It is markdown after all.  But that's for a different PR.